storm-core/src/jvm/org/apache/storm/pacemaker/PacemakerClientHandler.java

```diff
@@ -69,7 +69,7 @@ else if(evm instanceof HBMessage) {
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent event) {
-        LOG.error("Connection to pacemaker failed", event.getCause());
+        LOG.error("Connection to pacemaker failed. Trying to reconnect {}", event.getCause().getMessage());
```

Comment:
I don't think we want the stack. It ends up absolutely filling Nimbus logs with relatively useless messages when a Pacemaker node goes down. I'm open to ideas, though, because a trace for an NPE or something would be nice.